sql-database/transparent-data-encryption/setup-tde-byok-sqlmi.ps1

```diff
@@ -35,15 +33,17 @@ Update-AzKeyVaultNetworkRuleSet -VaultName "MyKeyVault" -DefaultAction Deny
 # $key = Add-AzureKeyVaultKey -VaultName MyKeyVault -Name MyTDEKey -Destination Software -Size 2048
 
 # Alternatively, the recommended way is to import an existing key from a .pfx file:
-$securepfxpwd = ConvertTo-SecureString -String 'MyPa$$w0rd' -AsPlainText -Force 
-$key = Add-AzureKeyVaultKey -VaultName 'MyKeyVault' -Name 'MyTDEKey' -KeyFilePath 'c:\some_path\mytdekey.pfx' -KeyFilePassword $securepfxpwd
+$securepfxpwd = ConvertTo-SecureString -String "MyPa$$w0rd" -AsPlainText -Force 
+$key = Add-AzKeyVaultKey -VaultName "MyKeyVault" -Name "MyTDEKey" -KeyFilePath "c:\some_path\mytdekey.pfx" -KeyFilePassword $securepfxpwd
```

Comment:
Key vault names are required to be universally unique. Either take a user-supplied value, or guarantee a unique name here.

Reply:
Added
![image](https://user-images.githubusercontent.com/40007119/55645381-44ee8b80-578d-11e9-994d-fdb05d2f1489.png)

Reply:
Since the AKV is added at the top, we moved this comment above:
![image](https://user-images.githubusercontent.com/40007119/55656698-d28da380-57ac-11e9-8237-785d766b96c8.png)